mlflow/pyfunc/model.py

```diff
@@ -659,12 +689,24 @@ def predict_stream(self, model_input, params: Optional[dict[str, Any]] = None):
         Returns:
             Streaming predictions.
         """
-        if inspect.signature(self.python_model.predict_stream).parameters.get("params"):
+        parameters = inspect.signature(self.python_model.predict_stream).parameters
+        kwargs = {}
+        if "params" in parameters:
+            kwargs["params"] = params
+        else:
+            _log_warning_if_params_not_in_predict_signature(_logger, params)
+        if (
+            "context" in parameters
+            or len([param for param in parameters if param != "params"]) == 2
+        ):
+            _logger.info(
+                _DROP_CONTEXT_IN_PYTHON_MODEL_PREDICT_INFO.format(func_name="predict_stream")
+            )
```

Comment:
It's possible that `context` is used, right?

Reply:
It's possible

Reply:
Do we need to show this message?

Reply:
It's actually possible to check if context is used.

Reply:
By inspecting user's predict function?

Reply:
> Do we need to show this message?

Maybe we don't need to. I think maybe in the future we can move context into property, so users can access it with self.context 🤔 Let me follow up on this plan, and we can raise a migration warning before we do this

Reply:
Do you think we can update PythonModel's predict function to `predict(model_input, params=None)` directly? Users who use `context` can continue to work since they're overriding the method anyways

Reply:
> By inspecting user's predict function?

We parse the function source code and check if `context` is used, but I don't think we need to do that. As we're not sure if it's used or not, we shouldn't warn a message.

Reply:
Why do we need to show this message? Can we just document it?

Reply:
Do we want to encourage users to drop it?

Reply:
because we'll remove it in the future?

Reply:
I think so. This is actually blocking some local testing, and lots of customers don't even know what this is for.
Maybe we shouldn't encourage them to drop it now, since we're not dropping it in our abstractmethod. 
The point is we need a way to tell customers that they should feel safe to drop the context parameter in "predict" function, maybe we can move it inside model logging instead of here?